lambdas/services/bulk_upload_service.py

```diff
@@ -115,25 +113,25 @@ def handle_sqs_message(self, message: dict):
             patient_ods_code = (
                 pds_patient_details.get_ods_code_or_inactive_status_for_gp()
             )
-
+            accepted_reason = None
             is_name_validation_based_on_historic_name = (
                 validate_filename_with_patient_details(file_names, pds_patient_details)
             )
-
+            if is_name_validation_based_on_historic_name:
+                accepted_reason = "Patient matched on historical name"
             if not allowed_to_ingest_ods_code(patient_ods_code):
                 raise LGInvalidFilesException("Patient not registered at your practice")
             patient_death_notification_status = (
                 pds_patient_details.get_death_notification_status()
             )
-            if is_formally_deceased(patient_death_notification_status):
-                raise PatientDeceasedException(
+            if patient_death_notification_status:
+                accepted_reason = (
                     f"Patient is deceased - {patient_death_notification_status.name}"
                 )
```

Comment:
If both `is_name_validation_based_on_historic_name` and `patient_death_notification_status` are true, the `accepted_reason` value assigned on Line 121 would be overwritten by Line 128. Would this pose a problem with what is saved to dynamo?

Reply:
Not sure what should take priority, let me check